udata/templates/dataset/display.html

```diff
@@ -370,6 +370,19 @@ <h3 class="text-left">{{ _('Producer') }}</h3>
                     </div>
                 </div>
                 {% endif %}
+
+                {# Territories panel #}
+                {% if dataset.spatial.handled_zones %}
+                <div class="panel panel-default">
+                    <div class="panel-body">
+                        <h3>{{ _('Territories')}}</h3>
+                        {% for territory in dataset.spatial.handled_zones %}
+                            <img src="{{ territory.logo_url(external=True) or theme_static('img/placeholder_territory.png', external=True) }}" alt="{{ territory.name }}" class="float-left" width="13px" /> <a href="{{ territory.url }}" title="{{ territory.name }}">{{ territory.name }}</a>{% if not loop.last %}, {% endif %}
```

Comment:
I did that because the placeholder comes from the theme, is there a way to deal with that case? There are placeholders in both the theme and the core which is confusing :/